README.md

```diff
@@ -53,13 +53,16 @@ When you open an issue please provide:
 - version of Vertica
 - smallest possible snippet of code to reproduce the problem
 
+## Examples
```

Comment:
Is the main readme the one that will show on npm, or the readme for packages/vertica-nodejs?

Reply:
I forgot about this. I think I do need to update the other readmes